src/js/components/ChatControl/ChatControl.js

```diff
@@ -0,0 +1,38 @@
+import PropTypes from 'prop-types';
+import React, { useContext } from 'react';
+import '../../../scss/components/ChatControl/ChatControl.scss';
+import { GlobalContext } from '../../contexts/GlobalContextWrapper';
+import ChatView from '../ChatView/ChatView';
+
+const ChatControl = (props) => {
+  const context = useContext(GlobalContext);
+  const { match } = props;
+  const { Config } = context;
+  const { groupNavDMButtons } = Config.get(['groupNavDMButtons']);
+  const button = groupNavDMButtons.filter((e) => e.id === match.params.id).shift();
+
+  if (!button) {
+    return (
```

Comment:
If this component is concerned with "control", should it have markup? Or can this jsx be moved to a component with a more declarative name?